src/graphic/shape/Rect.js

```diff
@@ -58,5 +58,13 @@ export default Path.extend({
         }
         ctx.closePath();
         return;
+    },
+
+    highlight: function () {
```

Comment:
There is no the concept of highlight/downplay/emphasis/normal in zrender yet.
I think it is not necessary to introduce those concept to zrender.